search2/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+use pyo3::prelude::*;
```

Comment:
Do we need this and main.rs at the moment?  If we're including these could we comment what we plan to implement in rust and what we're going to expose to python?

Reply:
Absolutely. Which option would you prefer; it may be a good exercise to move the save handlers business function (querying a chunk of data, and then writing the result to tsv chunk) in Rust. Small, easy, and a good exercise.

Reply:
It's probably worth a separate discussion about how we want to handle the rust rewrite.  Is any rust code load-bearing in this diff?  If not, would prefer to pull it for now.

Reply:
In the comment you're replying to it would be load bearing, yes

Reply:
To be clear, I'm not talking about a rewrite, rather the introduction of functions into lib.rs. This module is intended to serve as a basic test for Rust integration, so ripping out lib.rs (which Maturin probably wants around anyway), would kind of defeat the purpose

Reply:
ok, could we just add a comment then stating that it's a stub for future development?  lib.rs and main.rs are clearly stubs but the Cargo.toml has non-trivial dependencies, so a comment would help the reader to understand at a glance what the present state of things is.